src/MockClassifier.Api/Services/Dmr/DmrService.cs

```diff
@@ -40,19 +44,41 @@ public async Task ProcessRequestsAsync()
             {
                 try
                 {
-                    var response = await httpClient.PostAsJsonAsync("/", request).ConfigureAwait(true);
-                    _ = response.EnsureSuccessStatusCode();
+                    // Setup content
+                    var jsonPayload = JsonSerializer.Serialize(request.Payload);
+                    var jsonPayloadBase64 = _encodingService.EncodeBase64(jsonPayload);
+                    using var content = new StringContent(jsonPayloadBase64, Encoding.UTF8, MediaTypeNames.Application.Json);
+
+                    // Setup message
+                    using var requestMessage = CreateRequestMessage(request, content);
 
+                    // Send request
+                    var response = await _httpClient.SendAsync(requestMessage).ConfigureAwait(false);
+                    _ = response.EnsureSuccessStatusCode();
 
-                    // Not sure how to resolve rule CA1848 so removing logging for now
-                    //logger.LogInformation($"Callback to DMR. Ministry = {request.Payload.Ministry}, Messages = {string.Join(", ", request.Payload.Messages)}");
+                    _logger.DmrCallback(request.Payload.Classification, request.Payload.Message);
                 }
-                catch (HttpRequestException)
+                catch (HttpRequestException exception)
                 {
-                    // Not sure how to resolve rule CA1848 so removing logging for now
-                    //logger.LogError(exception, "Call to DMR Service failed");
```

Comment:
Now that the logger has been correctly setup, can we remove these comments and commented code? Applies throughout